test.scss

```diff
@@ -0,0 +1,20 @@
+.testing {
+  font-size: 16px;
+  text-align: left;
```

Comment:
Properties should be ordered font-size, position, right, text-align, top, z-index